Assets/Scripts/System/SetupSceneSystem.cs

```diff
@@ -36,6 +39,8 @@ public void SetupScene(string path)
             _cameraSystem.CameraStartup();
 
             _setupSceneEventListenersSystem.SetupSceneEventListeners();
-        }
+
+			_frameTimeSystem.SetApplicationTargetFramerate();
```

Comment:
You should put this in the `StartUpSystem`. Because it needs to be called once for the entire application and not for every scene